python/py-precis-i18n/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-precis-i18n
+version             1.0.4
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         PRECIS framework for Unicode
+long_description    Implementation of PRECIS framework for Unicode to be used with usernames & passwords
+homepage            https://pypi.org/project/precis-i18n
```

Comment:
please use https://github.com/byllyfish/precis_i18n

Reply:
@reneeotten Oh come on, I used it initially, and you told me to switch to PyPi: https://github.com/macports/macports-ports/pull/14782#discussion_r871951655